api/app/src/external/aws/s3.ts

```diff
@@ -0,0 +1,6 @@
+import * as AWS from "aws-sdk";
+import { Config } from "../../shared/config";
+
+export function makeS3Client() {
+  return new AWS.S3({ signatureVersion: "v4", region: Config.getAWSRegion() });
```

Comment:
It seems it never worked on staging: https://docs.aws.amazon.com/general/latest/gr/s3.html#s3-core

Reply:
Good catch - I haven't tested the doc download on Staging